mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -343,12 +343,9 @@ function resultsToTables(results, target, uri) {
                     }
                     newalbum = [];
                     newtlids = [];
-                    if (results[i].album) {
-                        coversList.push([results[i].album, i]);
-                    }
                 } //newalbum length
                 if (results[i].album) {
-                    coversList.push([results[i].album, i]);
+                    coversList.push([results[i].uri, i]);
```

Comment:
Oh, I didn't realise it was previously in *both* places. Surely once, inside the if statement should be sufficient. I agree it's a beast of a function and I've always steered clear but looking last night it's not too bad. It's just trying to split the results up by album so it can insert an album row (but only in the case where there are multiple consecutive  tracks for the album).